src/main/java/com/solutions/shop/repository/ICustomerRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.solutions.shop.repository;
+
+import com.solutions.shop.model.Customer;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ICustomerRepository extends JpaRepository<Customer, Integer> {
```

Comment:
Ask @jsdima maybe you can remove I letter in interface name start

Reply:
@jsdima, should I remove "I" letter in interface name?